agents/node/vcxagent-core/src/agent/service-ledger-creddef.js

```diff
@@ -0,0 +1,41 @@
+const { CredentialDef } = require('@absaoss/node-vcx-wrapper')
+
+module.exports.createServiceLedgerCredDef = function createServiceLedgerCredDef (logger, storeSchema, loadSchema, storeCredDef, loadCredDef, listCredDefIds) {
+  async function createCredentialDefinition (schemaId, credDefId) {
+    logger.info('Create a new credential definition on the ledger')
+    const data = {
+      name: credDefId,
+      paymentHandle: 0,
+      revocationDetails: {
+        supportRevocation: true,
+        tailsFile: '/tmp/tails',
+        maxCreds: 5
+      },
```

Comment:
It could be useful to have the revocation / `maxCreds` optional / selectable by the client code